README.md

```diff
@@ -61,13 +62,17 @@ This command will download the script from the specified URL and execute it with
 - --inventory <PATH>: Specify the path to the inventory file (required for --remote).
 - --ssh-key <SSH_KEY_FILE>: Specify the path to the SSH key file (required for --remote).
 - --user <USER>: Specify the user for running Ansible (required for --local).
+- --tags <TAGS>: Specify tags for Ansible playbook tasks (comma-separated, e.g., 'devops,shell,desktop').
+
+### Ansible Tags
+The tasks for installing `VLC` and `Telegram` have `desktop` tag, all other tools have `devops` tag and all shell utilities have `shell` tag. You can find their details at the [Tools Installed 🧰](#tools-installed-) section.
 
 ### Running in Local Mode with Custom DNS 📌
 If you want to run the playbook in local mode and set a custom DNS, you can use the following command:
 
 ---
 ```bash
-curl -sSL "https://raw.githubusercontent.com/data-burst/ubuntu_awesome_packages_installation/installation.sh" | bash -s -- --local --user <USER> --set-dns <DNS_IP>
+curl -sSL "https://raw.githubusercontent.com/data-burst/ubuntu_awesome_packages_installation/installation.sh" | bash -s -- --local --user <USER> --set-dns <DNS_IP> --tags <TAGS>
```

Comment:
Is this tag required to be defined or it is optional? In case of required please set a default value for it.

Reply:
If no tags are entered, I have parsed an error. Instead of setting a default tag, I prefer to allow the user to choose the tags he/she prefers.